spec/providers/coreIntegration/rtcpeerconnection.integration.src.js

```diff
@@ -40,6 +40,15 @@ module.exports = function (pc, dc, setup) {
       });
       alice.createOffer().then(function (offer) {
         return alice.setLocalDescription(offer).then(function () {return offer; });
+      }).then(function() {
+        // Delay 100ms for Firefox to get its act together.
+        return new Promise(function(resolve) {
+          setTimeout(function() {
+            alice.createOffer().then(function (offer) {
```

Comment:
`createOffer` is a const-method; it doesn't mutate the RTCPeerConnection.  However, it's not supposed to be necessary to call it multiple times (unless `.onnegotiationneeded` fires).  The only difference we've seen so far is the inclusion of more local candidates ... but those candidates should also be provided via `.onicecandidate`.

It should be safe, but it shouldn't be necessary.